crates/aptos/src/account/create.rs

```diff
@@ -31,7 +34,9 @@ pub struct CreateAccount {
     /// Flag for using faucet to create the account
     #[clap(long)]
     use_faucet: bool,
-
+    /// URL for the faucet
+    #[clap(long)]
+    faucet_url: Option<Url>,
```

Comment:
Hmm, should we just suggest in all the tutorials to run `aptos init` to start?

And not worry about adding these options to different commands. Just use the default if they haven't run `aptos init`

Reply:
We should, I think we're getting finally to the point we can replace all the other tools.